debian/control

```diff
@@ -2,7 +2,11 @@ Source: vncdotool
 Maintainer: Marc Sibson <sibson+vncdotool@gmail.com>
 Section: python
 Priority: optional
-Build-Depends: python-setuptools (>= 0.6b3), debhelper (>= 7), python-support (>= 0.8.4)
+Build-Depends:
+ python-setuptools (>= 0.6b3),
+ debhelper (>= 7),
+ python-support (>= 0.8.4),
+ python-sphinx
```

Comment:
that's for checking that.  Sorry, I'm out of touch with packaging.  Does the package use the built docs? Or is debhelper running the make command simply because it detected a makefile present?  I've created a new [issue](https://github.com/sibson/vncdotool/issues/82) to track if we should change the packaging to exclude the docs and allow this PR to be merged now.